src/test/java/spoon/test/position/PositionTest.java

```diff
@@ -424,6 +426,13 @@ public void testPositionGeneric() throws Exception {
 		assertEquals("protected static", contentAtPosition(classContent, position3.getModifierSourceStart(), position3.getModifierSourceEnd()));
 	}
 
+	@Test(timeout=10000)
```

Comment:
is there a reason for the timeout?

Reply:
The test specifically tests to make sure PositionBuilder terminates in a reasonable time. The existing bug caused non-termination, therefore if it doesn't terminate in a reasonable time, we assume it to be non-terminating